public/Sjofn/homework3.html

```diff
@@ -0,0 +1,100 @@
+            Crownvetch is toxic to horses and other non-ruminants because of the presence of nitroglycosides. If consumed in large amounts, it can cause slow growth, paralysis, or even death. However, this is not true for ruminant animals such as cattle, goats, and sheep. These aliphatic nitro compounds are degraded in ruminant digestion and do not affect the animals.</td>
+    </tr>
+    <tr>
+        <td>V</td>
+        <td>Datura</td>
+        <td><img src="./assets/datura.jpg"></td>
+        <td>Datura is a genus of nine species of poisonous Vespertine flowering plants belonging to the family Solanaceae. They are commonly known as daturas, but also known as devil's trumpets[1] (not to be confused with angel's trumpets, which are contained in the closely related genus Brugmansia). Other English common names include moonflower, jimsonweed, devil's weed, hell's bells and thorn-apple. The Mexican common names Toloache and Tolguacha derive from the Nahuatl name Tolohuaxihuitl meaning "the plant with the nodding head" (in reference to the nodding seed capsules of Datura species belonging to section Dutra of the genus). Datura species are native to dry, temperate, and subtropical regions of the Americas and are distributed mostly in Mexico, which is considered the center of its origin. Although Datura ferox occurs in Argentina, Datura metel in Asia, and Datura leichardthii in Australia, these species were probably introduced from Mexico.[2] Its distribution within the Americas and North Africa, however, is most likely restricted to the United States, Mexico and Southern Canada in North America, and Tunisia in Africa where the highest species diversity occurs.
+            All species of Datura are poisonous, especially their seeds and flowers which can cause respiratory depression, arrhythmias, hallucinations, psychosis, as well as death if taken internally.</td>
+    </tr>
+    <tr>
+        <td>VI</td>
+        <td>Red magic lily</td>
+        <td><img src="./assets/redmagiclily.jpg"></td>
+        <td>Lycoris radiata, known as red spider lily, hell flower, red magic lily, or equinox flower, is a plant in the amaryllis family, Amaryllidaceae, subfamily Amaryllidoideae. Originally from China, Korea, and Nepal, it was introduced into Japan and from there to the United States and elsewhere. It is considered naturalized in Seychelles and in the Ryukyu Islands.[3] It flowers in the late summer or autumn, often in response to heavy rainfall. The common name hurricane lily refers to this characteristic, as do other common names, such as resurrection lily; these may be used for the genus as a whole.</td>
+    </tr>
+    <tr>
+        <td>VII</td>
+        <td>Wisteria</td>
+        <td><img src="./assets/wisteria.jpg"></td>
```

Comment:
You forgot to add this image to VCS (Version Control System)

Reply:
![wisteria_missing_image](https://user-images.githubusercontent.com/16742190/74083115-291cdb80-4a69-11ea-90c8-0ab82aeb5c6c.JPG)